docs/src/environments/_index.md

```diff
@@ -54,106 +89,91 @@ Each new environment you create is considered a **child** of the **parent** envi
 
 Each child environment can [sync](../other/glossary.md#sync) code and/or data down from its parent
 and [merge](../other/glossary.md#merge) code up to its parent.
-The child environments are used for development, staging, and testing.
+You can use child environments for development, staging, and testing.
 
 When you [branch](../other/glossary.md#branch) an environment to create a new child environment,
 its parent is the environment it was created from.
 If you push a branch through Git or a [source integration](../integrations/source/_index.md),
-the parent defaults to the default environment.
+the parent environment is your [default environment](#default-environment).
 
 You can always [change an environment's parent](./change-parent.md).
 
-## Workflows
+### Workflows
+
+Since you can customize your [environment hierarchy](#hierarchy), you can create your own workflows. 
+You don't have to follow any rules when branching environments.
```

Comment:
:hammer_and_wrench: I think your wording is clearer than the original, but I'm not really sure the sentence adds much. Unless you seem something useful I'm missing.
```suggestion
```